interface-manager/src/interface/mod.rs

```diff
@@ -11,6 +11,7 @@ mod tap;
 mod vrf;
 mod vtep;
 
+
```

Comment:
Nit: leftover blank line

Reply:
I see you fix it in the last commit. Consider squashing with `feat(interface-manager)!: netdevsim part 1`.